File: Rust/src/atcoder/abc240/a_edge_checker.rs
pub fn run(a: i8, b: i8) -> String {
    if (a - b).abs() == 1 || (a - b).abs() == 9 {
        String::from("Yes")
    } else {
        String::from("No")
    }
}

#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn test() {
        assert_eq!(String::from("Yes"), run(2, 3));
        assert_eq!(String::from("No"), run(3, 5));
        assert_eq!(String::from("Yes"), run(1, 10));
    }
}

File: Erlang/AtCoder/abc240/a_edge_checker.erl
-module(a_edge_checker).
-export([main/0]).

% AとBの絶対値を計算し、値が1か9ならYesを返し、そうでないならNoを返す
a_edge_checker(A, B) ->
  case (abs (A - B)) of
    1 -> "Yes";
    9 -> "Yes";
    _ -> "No"
  end.

main() ->
  io:format("~p ~n", [a_edge_checker(1, 10) =:= "Yes"]),
  io:format("~p ~n", [a_edge_checker(5, 4) =:= "Yes"]),
  io:format("~p ~n", [a_edge_checker(5, 1) =:= "No"]).
